apps/meteor/client/views/invite/InvitePage.tsx

```diff
@@ -28,7 +30,8 @@ const InvitePage = (): ReactElement => {
 		return <PageLoading />;
 	}
 
-	if (isValidInvite) {
+	if (isValidInvite && token) {
+		localStorage.setItem(KEY, token);
```

Comment:
invite token should be removed on successful non-SAML login as well.

Possibly needs to also be removed on failed login as well?

Reply:
Hmmm am I not covering both cases [here](https://github.com/RocketChat/Rocket.Chat/pull/34873/files#diff-450ab68d9ab08bab95e5c2a54d94e7e30bd6be6d1939dde254842be4eca8605d) already? Real question bc I thought so, but I might me missing somethind
That mutation is always called since this is responsible for redirecting the user to the invite link